### DISCUSSION TITLE
Pre-trained Models in Dropbox

### DISCUSSION THREAD

**Discussion by JaninaMattes** [hpanwar08/detectron2#49] [Q&A]
Hi @hpanwar08 thank you very much for the pre-trained models in Detectron2. Is there also a model.pth file for the Mask R-CNN resnext101_32x8d architecture? I could not find it in the dropbox folder. 

Thanks in advance!

![image](https://user-images.githubusercontent.com/30773447/122922035-81553480-d363-11eb-9d6d-e1f2c1351ff9.png)

**Comment by hpanwar08** [comment]
Hi @JaninaMattes, there are only trimmed files in dropbox.

**Reply by JaninaMattes** [reply]
Hi @hpanwar08 thanks for the quick response. What is the difference between trimmed and not trimmed .pth model files? Do I need to follow any specific steps when loading the trimmed models?

**Reply by hpanwar08** [reply]
Refer https://github.com/hpanwar08/detectron2/issues/19
